pycaret/internal/pycaret_experiment/ts_unsupervised_experiment.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+
+from pycaret.internal.pycaret_experiment.supervised_experiment import (
+    _SupervisedExperiment,
+)
+from pycaret.internal.pycaret_experiment.unsupervised_experiment import _UnsupervisedExperiment
+
+
+class _TSUnsupervisedExperiment(_UnsupervisedExperiment):
+    """TODO: We'll need to refactor the code below to be useful for generic unsupervised experiments one
```

Comment:
Agreed, the point is that any common aspects can be upstreamed into the parent class and any specificities can be implemented in the child class as necessary.